### ISSUE TITLE
links to doxygen are broken on project "master pages" (Trac #2045)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2037]
The link at the bottom of http://software.icecube.wisc.edu/documentation/projects/icetray/index.html leads to a 404. The correct URL is http://software.icecube.wisc.edu/documentation/cpp/icetray.html

Migrated from https://code.icecube.wisc.edu/ticket/2045
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T23:44:46", 
    "description": "The link at the bottom of http://software.icecube.wisc.edu/documentation/projects/icetray/index.html leads to a 404. The correct URL is http://software.icecube.wisc.edu/documentation/cpp/icetray.html", 
    "reporter": "dsoldin", 
    "cc": "nega, olivas", 
    "resolution": "duplicate", 
    "_ts": "1547250286355415", 
    "component": "combo core", 
    "summary": "links to doxygen are broken on project \"master pages\"", 
    "priority": "normal", 
    "keywords": "documentation", 
    "time": "2017-06-29T15:34:05", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170629T15:39:40: nega changed cc from "nega" to "nega, olivas"

**Comment by icecube-trac** [comment]
Trac update at 20170629T15:44:17: 
* olivas changed status from "new" to "assigned"
* olivas commented:

> That link works just fine for me.  I'm guessing you were accessing this while the docs were building.  Seems like it should be easy to build in a tmp dir and swap docs after a successful build.


* olivas changed owner from "kjmeagher" to "nega"

**Comment by icecube-trac** [comment]
Trac update at 20170629T15:47:55: 
* nega commented:

> no. i did it right now. want video?
> 


* nega changed owner from "nega" to "kjmeagher"

**Comment by icecube-trac** [comment]
Trac update at 20170629T15:57:05: 
* nega commented:

> The link is indeed broken. You can even tell by looking at the [source:IceTray/projects/icetray/trunk/resources/docs/index.rst source file]. Since this is a project docs issue it should go to the project owner.


* nega changed owner from "kjmeagher" to "olivas"

**Comment by icecube-trac** [comment]
Trac update at 20170629T17:59:15: 
* olivas changed status from "assigned" to "accepted"
* olivas commented:

> My mistake.  I read that too quickly and assumed the link to icetray docs was being reported as broken.
>

**Comment by icecube-trac** [comment]
Trac update at 20170629T18:03:31: 
* nega changed status from "accepted" to "closed"
* nega changed resolution from "" to "duplicate"
* nega commented:

> "Duped" in #2039

**Comment by icecube-trac** [comment]
Trac update at 20170706T14:55:41: nega commented:

> In changeset 2939:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2939"
> fix broken links to the doxygen docs
> 
> 1 commit enter. 200 broken links leave.
> or something.
> 
> this fixes most of the broken links to "/doxygen/".
> 
> there's still broken links. it would be a good idea to run all the docs
> through validator.w3.org and clean up those links and links to permenantly
> redirected urls.
> 
> see #2037
> closes #2039
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190111T23:44:46: nwhitehorn commented:

> In changeset 170014:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170014"
> fix broken links to the doxygen docs
> 
> 1 commit enter. 200 broken links leave.
> or something.
> 
> this fixes most of the broken links to "/doxygen/".
> 
> there's still broken links. it would be a good idea to run all the docs
> through validator.w3.org and clean up those links and links to permenantly
> redirected urls.
> 
> see #2037
> closes #2039
> ```
